frontend/src/components/dashboard/create/common/BodyTemplate/BodyTemplate.tsx

```diff
@@ -11,40 +12,51 @@ import {
   StepSection,
 } from 'components/common'
 import SaveDraftModal from 'components/dashboard/create/save-draft-modal'
-import { exceedsCharacterThreshold, saveTemplate } from 'services/sms.service'
+import { saveTemplate } from 'services/sms.service'
```

Comment:
This component does not work with Telegram.

Reply:
Ah my bad. I guess this is a case where frontend tests would've caught the error haha